Source/Core/Core/FifoPlayer/FifoPlayer.cpp

```diff
@@ -348,6 +348,19 @@ void FifoPlayer::SetupFifo()
 
 void FifoPlayer::LoadMemory()
 {
+	UReg_MSR newMSR;
+	newMSR.DR = 1;
+	newMSR.IR = 1;
+	MSR = newMSR.Hex;
+	PowerPC::ppcState.spr[SPR_IBAT0U] = 0x80001fff;
+	PowerPC::ppcState.spr[SPR_IBAT0L] = 0x00000002;
+	PowerPC::ppcState.spr[SPR_DBAT0U] = 0x80001fff;
+	PowerPC::ppcState.spr[SPR_DBAT0L] = 0x00000002;
+	PowerPC::ppcState.spr[SPR_DBAT1U] = 0xc0001fff;
+	PowerPC::ppcState.spr[SPR_DBAT1L] = 0x0000002a;
```

Comment:
Is this right for Wii? Does it matter? Do we have to do anything for games that use weird BATs, i.e. save the BAT setup in the FIFO?